docs/model_monitoring/model-monitoring-deployment.ipynb

```diff
@@ -257,36 +257,13 @@
     "If you do not have a Grafana service running,\n",
     "see <a href=\"https://www.iguazio.com/docs/latest-release/services/fundamentals/#create-new-service\" target=\"_blank\">Creating a New Service</a> to create and configure it.\n",
     "\n",
-    "1. Make sure you have the `model-monitoring` as a Grafana data source configured in your Grafana service. If not,\n",
-    "add it by:\n",
+    "Add access keys to your model-monitoring data source by:\n
     "   1. Open your grafana service.\n",
     "   2. Navigate to `Configuration -> Data Sources`.\n",
-    "   3. Press `Add data source`.\n",
-    "   4. Select the `SimpleJson` datasource and configure the following parameters.\n",
-    "       ```Name: model-monitoring\n",
-    "       URL: http://mlrun-api:8080/api/grafana-proxy/model-endpoints\n",
-    "       Access: Server (default)\n",
-    "\n",
-    "       ## Add a custom header of:\n",
-    "       if working with Iguazio 3.0.x:\n",
-    "           X-V3io-Session-Key: <YOUR ACCESS KEY>\n",
-    "       if working with Iguazio 3.2.x:\n",
-    "           cookie: session=j:{\"sid\": \"<YOUR ACCESS KEY>\"}\n",
-    "       ```\n",
-    "   5. Press `Save & Test` for verification. You will receive a confirmation with either a success, or a failure message.\n",
-    "\n",
-    "2. Download the following monitoring dashboards:\n",
-    "    * {download}`Model Monitoring - Overview <./dashboards/model-monitoring-overview.json>`\n",
-    "    * {download}`Model Monitoring - Details <./dashboards/model-monitoring-details.json>`\n",
-    "    * {download}`Model Monitoring - Performance <./dashboards/model-monitoring-performance.json>`\n",
-    "\n",
-    "3. Import the downloaded dashboards to your Grafana service.\n",
-    "To import that dashboards into your Grafana service:\n",
-    "    1. Navigate to your Grafana service in the Services list and press on it\n",
-    "    2. Press the dashboards icon in left menu\n",
-    "    3. In the dashboard management screen press the IMPORT button, and select one file to import. Repeat this step for each dashboard.\n",
-    "\n",
+    "   3. Press `mlrun-model-monitoring`.\n",
+    "   4. In Custom HTTP Headers configure the cookie parameter. Value should be `session=j:{\"sid\": \"<YOUR ACCESS KEY>\"}`\n",
```

Comment:
Correct to: In Custom HTTP Headers, configure the cookie parameter. Set the value to `session=j:{\"sid\": \"<YOUR ACCESS KEY>\"}`\n",